sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/common/ChangeFeedMode.java

```diff
@@ -3,12 +3,9 @@
 
 package com.azure.cosmos.implementation.changefeed.common;
 
-import com.azure.cosmos.util.Beta;
-
 /**
  * Change feed mode
  */
-@Beta(value = Beta.SinceVersion.V4_35_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
 public enum ChangeFeedMode {
```

Comment:
same here, please revert this.

Reply:
Implementation package - no Beta needed.